bot/cogs/metrics.py

```diff
@@ -71,6 +76,27 @@ def __init__(self, bot: Bot) -> None:
             guild_id=message.guild.id,
         ).inc()
 
+    @Cog.listener()
+    async def on_command_completion(self, ctx: Context) -> None:
+        """Increment the command completion counter."""
+        if ctx.message.guild is not None:
+            path = []
+            if (
```

Comment:
This doesn't handle enough cases, why not use [this](https://discordpy.readthedocs.io/en/latest/ext/commands/api.html#discord.ext.commands.Command.parents)?

Reply:
Cases I mean are like `!a b c` or deeper.

Reply:
b0b9cdd